lib/resolver/javascript-universal.js

```diff
@@ -3,6 +3,13 @@ import javascriptFile from './javascript-file.js';
 import liveResolverQuery from './live-resolver-query.js';
 import liveResolverPing from './live-resolver-ping.js';
 
+function getTopModuleName(target) {
+  const isScoped = target.startsWith('@');
```

Comment:
Hmm, I'm not too familiar with other package managers. In my opinion, this is at least an incremental improvement upon the current state of things. If we later discover that other edge cases are not handled correctly, we can fix those then. Does that sound ok to you?